components/applications-service/pkg/storage/postgres/services.go

```diff
@@ -90,6 +90,33 @@ LEFT JOIN supervisor AS sup
 ORDER BY %s %s
 LIMIT $1
 OFFSET $2
+`
+	selectDisconnectedServices = `
+SELECT s.id
+  , s.origin AS origin
+  , s.name AS name
+  , s.version AS version
+  , s.release AS release
+  , s.status AS status
+  , s.health AS health
+  , sg.name AS group
+  , d.app_name AS application
+  , d.environment AS environment
+  , sup.member_id AS sup_member_id
+  , sup.fqdn AS fqdn
+  , s.channel as channel
+  , sup.site as site
+  , s.last_event_occurred_at as last_event_occurred_at
+  , s.previous_health as previous_health
+  , s.health_updated_at as health_updated_at
+FROM service AS s
+LEFT JOIN service_group AS sg
+  ON s.group_id = sg.id
+LEFT JOIN deployment AS d
+  ON s.deployment_id = d.id
+LEFT JOIN supervisor AS sup
+  ON s.sup_id = sup.id
+WHERE last_event_occurred_at < now() - interval '%d minutes'
```

Comment:
💭 I wonder, does `($1 || ' minutes')::interval` work here?  That would allow you to use positional arguments rather than an Sprintf.  Although, in this case I'm not too worried about escaping issues so this is fine as well.

Reply:
I tried with `$1` and didn't work, I can try your solution and report back! 👍

Reply:
Cool, no worries if not, I didn't test it out, so I'm not sure it'll work.

Reply:
It worked your way Steven! Thanks for your feedback. 💯